src/Get.tsx

```diff
@@ -297,7 +307,9 @@ class ContextlessGet<TData, TError> extends React.Component<
  * in order to provide new `parentPath` props that contain
  * a segment of the path, creating composable URLs.
  */
-function Get<TData = any, TError = any>(props: GetProps<TData, TError>) {
+function Get<TData = any, TError = any, TQueryParams = { [key: string]: any }>(
```

Comment:
Can we use `Record<>` instead of `{ [key: string]: any }` in this case?

Reply:
`Record` of what? `Record` is for describe a record of an object, here it's a default value that is a dictionnary, so `{[key: string]: any}`

```ts
/**
 * Construct a type with a set of properties K of type T
 */
type Record<K extends keyof any, T> = {
    [P in K]: T;
};
```